package.json

```diff
@@ -1,24 +1,24 @@
 {
```

Comment:
I think there is a diff in your prettier config here, not sure why this file changed.

Reply:
Yah, I've resolved the issue, thanks.